### PR TITLE
Use ruby command "gem list" for foreman existence test

### PR DESCRIPTION
**Pull request by elalemanyo**
Hi,
I know this does not solve any current problem, but maybe would be better to use the command [`gem list`](https://guides.rubygems.org/command-reference/#gem-list) to check for foreman.  This command gives a better output (boolean) and appropriate exit code.

Thanks

Related with rails/cssbundling-rails/pull/91

### CODE REVIEW

**Comment by elalemanyo** [comment]
@MrJoy I know this is something you fixed a few days ago, but maybe this is a good way to improve it, what do you think?
Or is your way better?
Thanks

**Comment by kevynlebouille** [comment]
Hi @elalemanyo ,

Your version output true or false in the console.

Here is my proposal

```
if ! gem list foreman -i --silent; then
```

File: lib/install/dev [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-if ! foreman version &> /dev/null
-then
+if ! gem list -i foreman; then
```
Comment (elalemanyo) [inline lib/install/dev:3]:
```suggestion
if ! gem list foreman -i --silent; then
```

**Comment by elalemanyo** [comment]
@MrJoy do you think we can merge this? 😁

**Comment by MrJoy** [comment]
I'm not a maintainer, so it's not really for me to say.  I can say that the fix works perfectly fine for my use-case and I fully support merging it.

**Pull request merged**